keps/sig-node/20181106-in-place-update-of-pod-resources.md

```diff
@@ -215,13 +270,38 @@ new desired resources (i.e Spec.Containers[i].Resources.Requests) to the sum.
   further action is taken.
   - Kubelet retries the Pod resize at a later time.
 
+<<[UNRESOLVED]>>
+This is close to what I wrote above, but still vague.  What does it mean to
+"reject" a resize?  How do controllers know it has been deferred temporarily
```

Comment:
This should be worded better - reject comes across as a negative affirmation, whereas it really means that an internal function canAdmitPod says "no" for new desired resources.

The outcome to the user is they won't see Resources == ResourcesAllocated until kubelet retries and it goes through at a later point when resources become available - we retry indefinitely. 

At this time, we expect VPA (consumer of this feature) to set a timeout if user has a policy allowing pod eviction + recreate with new resources in the event that resize does not happen for a certain duration.

Reply:
So the UX is "radio silence"?  That seems wrong.